pkgs/development/interpreters/python/cpython/default.nix

```
@@ -518,7 +532,7 @@ in with passthru; stdenv.mkDerivation {
       high level dynamic data types.
     '';
     license = licenses.psfl;
-    platforms = with platforms; linux ++ darwin;
+    platforms = platforms.linux ++ platforms.darwin;
```

```suggestion
    platforms = platforms.linux ++ platforms.darwin;
```
Why not `platforms.unix`?